.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+orbs:
+  python: circleci/python@2.0.3
+  tmh-blackduck: mobilityhouse/blackduck-runner-orb@1
+
+aliases:
+  - &base_blackduck_main
+    name: "blackduck automatic"
+    context:
+      - tmh
```

Comment:
@FireyFly I think the `tmh` context here is not needed, as the tool does not have any private dependencies

Reply:
You're right, removed that context since it's superfluous.